plugins.json

```diff
@@ -1,4 +1,91 @@
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.4.1",
+    "hosting": "",
+    "author_type": "mattermost",
+    "release_stage": "beta",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmIqUGIACgkQ0bVLR6XO/sT/Ng//dtmR9PxyG3DBOAbq+6hQ2lMoNg8GLd1lkwV8ytmykwlfUUbJtJaItx1OsJQsHQ/RzftNW6rsuV4MqF70ZV/sM0l4ggb96Lry97yHOsD2Fdmn12JlRtdlVWVR5M3D6tvVYVuHkr/V48ZpQ9poTPTmjQqaDJctxiBt3KBypH3KADiWMiEhzjX5qaVPKiEVObQDVGCQynrEXlYWxMYQQYNhBk3QcUlqXqZscEuePsEK7bkxVIznFSNZ6MyMUaqSFHuJbAN6pnG1HmiHYjrvtozlrueXW8NLvGkp6nVXmW1kd38TCtKgRyl3DEMleuicae9UpSYdSWgP2HgSMBG4oujRxcIRQ+pcDK2mTepf2TX63R61BAWxCeT1uCtYQXAG0iIay7HQdIbmp8pFy+kniTVolMXHxN5sYiVqruEencx0uFNprEgOvtCDax2B/SZF1Jwr/QjrK1zE6SF1HxUlYUCQFgMflbqGkpWGB3XcDxWmZHluKwv8w28HfCzlAuhfm2vWxtUHOcFrrb85OwaiQCPAQm8CZv6YDssqfQcQkn29LV8usW0bQNb9Q4OYQlDB0DDusFPJhmYIfca82F1QoiKt2865U+1yJ6zOMsn3GSBIJ599l8anpQt6KMLh9CWut7yi60sMSpjBF2NriRORgirfjHJRV8jAqzW8yqqIbJbdMZA=",
+    "repo_name": "mattermost-plugin-calls",
+    "manifest": {
+      "id": "com.mattermost.calls",
+      "name": "Calls",
+      "description": "Integrates real-time voice communication in Mattermost",
+      "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+      "support_url": "https://github.com/mattermost/mattermost-plugin-calls/issues",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.4.1",
+      "version": "0.4.1",
+      "min_server_version": "6.3.0",
+      "server": {
+        "executables": {
+          "linux-amd64": "server/dist/plugin-linux-amd64",
+          "darwin-amd64": "server/dist/plugin-darwin-amd64"
+        },
+        "executable": ""
+      },
+      "webapp": {
+        "bundle_path": "webapp/dist/main.js"
+      },
+      "settings_schema": {
+        "header": "",
+        "footer": "",
+        "settings": [
+          {
+            "key": "ICEHostOverride",
+            "display_name": "ICE Host Override",
+            "type": "text",
+            "help_text": "The IP (or hostname) to be used as the host ICE candidate. If empty, it defaults to resolving via STUN.",
+            "placeholder": "",
+            "default": ""
+          },
+          {
+            "key": "UDPServerPort",
+            "display_name": "RTC Server Port",
+            "type": "number",
+            "help_text": "The UDP port the RTC server will listen on.",
+            "placeholder": "8443",
+            "default": 8443
+          },
+          {
+            "key": "ICEServers",
+            "display_name": "ICE Servers",
+            "type": "text",
+            "help_text": "A comma separated list of ICE servers URLs (STUN/TURN) to use.",
+            "placeholder": "stun:example.com:3478",
+            "default": "stun:stun.l.google.com:19302,stun:global.stun.twilio.com:3478"
```

Comment:
This should've changed :thinking:

Reply:
@streamer45 It /has/ changed in the release package on github. I'll check to see why it didn't change here...